packages/ui/src/message-room-details-dialog/components/DeleteChannel.jsx

```diff
@@ -1,13 +1,51 @@
 import ChannelModal from "./ChannelModal";
 import styles from "./archive-channel.module.css";
-
+import axios from "axios";
+import { useEffect, useState } from "react";
+import "react-notifications/lib/notifications.css";
+import {
+  NotificationManager,
+  NotificationContainer
+} from "react-notifications";
 const DeleteChannel = ({ closeEdit }) => {
+  const room = window.location.href.split("/").at(6);
```

Comment:
I'm still against this method of getting the room id. The url is about to be shortened and this will not be a working solution anymore. Is there another way to get the ID?